torch_xla/csrc/ops/ops_lower_fn.cpp

```diff
@@ -598,6 +599,11 @@ torch_xla::XlaOpVector Take::Lower(LoweringContext* loctx) const {
 
 torch_xla::XlaOpVector Tan::Lower(LoweringContext* loctx) const {
   xla::XlaOp xla_input = loctx->GetOutputOp(operand(0));
+  if (xla::primitive_util::IsIntegralType(XlaHelpers::TypeOfXlaOp(xla_input))) {
```

Comment:
I think we also need to update https://github.com/pytorch/xla/blob/9794392a519119a4e4b6ddd35f737ed9dd64eece/torch_xla/csrc/ops/ops_xla_shape_fn.cpp#L680-L682. I don't think result will be int type.

Reply:
Done